tests/unit/src/kakarot/precompiles/test_ecadd.cairo

```diff
@@ -29,30 +31,119 @@ func test__ecadd_impl{
     // Given
     alloc_locals;
 
-    let input_word0 = Helpers.load_word(ECPOINT_BYTES_LEN, calldata);
-    let input_word1 = Helpers.load_word(ECPOINT_BYTES_LEN, calldata + ECPOINT_BYTES_LEN);
-    let input_word2 = Helpers.load_word(ECPOINT_BYTES_LEN, calldata + ECPOINT_BYTES_LEN * 2);
-    let input_word3 = Helpers.load_word(ECPOINT_BYTES_LEN, calldata + ECPOINT_BYTES_LEN * 3);
-    let x0: BigInt3 = Helpers.to_bigint(input_word0);
-    let y0: BigInt3 = Helpers.to_bigint(input_word1);
-    let x1: BigInt3 = Helpers.to_bigint(input_word2);
-    let y1: BigInt3 = Helpers.to_bigint(input_word3);
-
-    let point0 = EcPoint(x0, y0);
-    let point1 = EcPoint(x1, y1);
-    let (expected_point: EcPoint) = ec_add(point0, point1);
-    let expected_x = Helpers.bigint_to_felt(expected_point.x);
-    let expected_y = Helpers.bigint_to_felt(expected_point.y);
+    let x0: BigInt3 = Helpers.bytes32_to_bigint(calldata);
+    let y0: BigInt3 = Helpers.bytes32_to_bigint(calldata + G1POINT_BYTES_LEN);
+    let x1: BigInt3 = Helpers.bytes32_to_bigint(calldata + G1POINT_BYTES_LEN * 2);
+    let y1: BigInt3 = Helpers.bytes32_to_bigint(calldata + G1POINT_BYTES_LEN * 3);
 
     // When
-    let (output_len, output: felt*, gas_cost) = PrecompileEcAdd.run(
+    let point0: G1Point = G1Point(x0, y0);
+    let point1: G1Point = G1Point(x1, y1);
+    let (expected_point: G1Point) = ALT_BN128.ec_add(point0, point1);
```

Comment:
Here we're just testing that `Precompuile.ec_add` is the same as `alt_bn128.ec_add` but we miss a test for `alt_bn128.ec_add` right?

Reply:
actually the CI is failing because of this added test. I suspect a misformed bigint since the error is

```
FAILED tests/integration/bytecode/test_kakarot.py::TestKakarot::test_execute[Precompiles - EC_ADD - playground test case] - assert [991811005130...8415326638035] == [158405335741...3740070691524]
  At index 0 diff: 9918110051302171585080402603319702774565515993150576347155970296011118125764 != 158405335741503967067615289663786980848246198220868600867374997558769250359476531654134399353573892026331489124207861880946782971100028308433740070691524
  Left contains one more item: 136801517948995470...8064868415326638035
  Full diff:
    [
  -  158405335741503967067615289663786980848246198220868600867374997558769250359476531654134399353573892026331489124207861880946782971100028308433740070691524,
  +  9918110051302171585080402603319702774565515993150576347155970296011118125764,
  +  1368015179489954701390400359078579693043519447331113978918064868415326638035,
    ]
```

Reply:
Thanks, fixed